### ISSUE TITLE
Describe how "sharedSecret" may be used in WebDAV protocol

### ISSUE THREAD

**Issue by michielbdejong** [cs3org/OCM-API#63]
If you look at the `$settings` in https://github.com/owncloud/core/blob/e5e3da4ee8179055d479025404f7b70996d5fed3/lib/private/Http/Client/WebDavClientService.php#L90 you'll see something like:
```php
[
  'baseUri' => 'https://oc1.docker/public.php/webdav/',
  'userName' => 'DyQARczFGQI7V3S',
  'password' => '',
  'authType' => 1,
]
```
 (source: https://github.com/SURFnet/rd-sram-integration/issues/114#issuecomment-1461757673)

I have a strong feeling that this `userName` value is the `protocol.webDav.sharedSecret` value from https://github.com/cs3org/OCM-API/blame/c590a3d6c2c0388bb63b76c7c4c30cc9aed5f75d/spec.yaml#L224 (or "protocol**s**" - see #62)

**Comment by gmgigi96** [comment]
I agree that is difficult to understand how to use the `sharedSecret` property in the WebDAV protocol. There is the same problem in reva, and indeed this only works between two reva as the WebDAV endpoint is open and the `sharedSecret` is always empty. But it does not work with other implementations.

I have already raised the problem in https://github.com/cs3org/OCM-API/pull/57#issuecomment-1420346364
In my opinion we should spell out as well in the WebDAV options how to use the shared secret.

**Comment by glpatcern** [comment]
I leave the issue open as it is valid for OCM v1.0, but we agreed a new model in the upcoming OCM v1.1, which settles this. That's why I removed the `needs discussion` label.

**Comment by glpatcern** [comment]
Eventually closed by #72.
